src/components/content-tab-assistant.tsx

```diff
@@ -12,6 +14,32 @@ import { MessageThinking } from './assistant-thinking';
 import Button from './button';
 import { AssistantIcon } from './icons/assistant';
 import { MenuIcon } from './icons/menu';
+import '../assistant-markdown.css';
+
+const md = new Remarkable( {
+	highlight: function ( str, lang ) {
+		if ( lang && hljs.getLanguage( lang ) ) {
+			try {
+				return hljs.highlight( lang, str ).value;
+			} catch ( err ) {
+				console.error( `Error highlighting with language ${ lang }:`, err );
+			}
+		}
```

Comment:
This logic assumes that the endpoint returns fenced Markdown code blocks formatted with the language included for syntax highlighting.